sonar-dotnet-shared-library/src/test/java/org/sonar/plugins/dotnet/tests/WildcardPatternFileProviderTest.java

```diff
@@ -236,6 +236,17 @@ public void should_fail_with_parent_folder_access_after_wildcard() {
     listFiles(path("*", "..", "foo.txt"), tmp.getRoot());
   }
 
+  @Test
+  public void given_pattern_with_windows_folder_separator_listFiles_detects_wrong_pattern(){
```

Comment:
the pattern is with mixed folder separators, right? Because `tmpRoot` and `FileSeparator` will be Windows, and `_temp/**/*.trx` are UNIX?

and rather than "detects", it should be "supports", because from the test I cannot see the detection itself (there's no message "Pattern detected")